tests/integration/testlib/cmdlib.py

```diff
@@ -49,10 +49,13 @@ def exec_cmd(cmd, env=None, stdin=None):
     )
 
     out, err = p.communicate(stdin)
+    returncode = p.returncode
```

Comment:
Please call `Popen()` instead with `check=check_returncode` and IMHO rename `check_returncode` to `check`.

Reply:
Reading the subprocess documentation I am not able to see the `check` parameter in the `Popen()` constructor.

Reply:
Are you maybe looking into Python 2.7 docs? I see it in 3.6 and later: https://docs.python.org/3.6/library/subprocess.html

Reply:
Sorry, I am looking into `Popen()` and I am not able to find `check` here 
```
class subprocess.Popen(args, bufsize=-1, executable=None, stdin=None, stdout=None, stderr=None, preexec_fn=None, close_fds=True, shell=False, cwd=None, env=None, universal_newlines=False, startupinfo=None, creationflags=0, restore_signals=True, start_new_session=False, pass_fds=(), *, encoding=None, errors=None)
```

I will try it anyway, thanks!

Reply:
Fixed. We are using now subprocess.run(). Thanks!